### DISCUSSION TITLE
Top Navigation Bar

### DISCUSSION THREAD

**Discussion by rdaraujo** [Nerwyn/material-you-theme#48] [Q&A] [answered]
Is it possible to put the Navigation Bar to the top, instead of the bottom?

**Comment by Nerwyn** [comment, marked as answer]
Not cleanly. You'd have to fork/modify the theme to do so. The CSS to do so is found [here](https://github.com/Nerwyn/material-rounded-theme/blob/2.1.10/src/common/root.css#L5-L6), and the instructions for compiling the theme are found [here](https://github.com/Nerwyn/material-rounded-theme?tab=readme-ov-file#developing-modifying-and-building-the-theme) (requires Python). There's some additional CSS you'd have to modify to merge the header and footer back together, which you'd have to dig around to change/undo.
